common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
 {
   "key.categories.voicechat": "Voice Chat",
-  "key.push_to_talk": "Push To Talk",
+  "key.push_to_talk": "Push to Talk",
   "key.whisper": "Whisper",
   "key.mute_microphone": "Mute Microphone",
-  "key.disable_voice_chat": "Disable Voice Chat",
+  "key.disable_voice_chat": "Disable Voice Chat Sounds",
   "key.hide_icons": "Hide Voice Chat Icons",
   "key.voice_chat_settings": "Voice Chat Settings",
-  "key.voice_chat": "Voice Chat GUI",
-  "key.voice_chat_group": "Voice Chat Group",
+  "key.voice_chat": "Voice Chat Menu",
+  "key.voice_chat_group": "Group Management",
   "key.voice_chat_toggle_recording": "Toggle Recording",
   "key.voice_chat_adjust_volumes": "Adjust Volumes",
   "gui.voicechat.voice_chat.title": "Voice Chat",
   "gui.voicechat.voice_chat_settings.title": "Voice Chat Settings",
   "gui.voicechat.adjust_volume.title": "Adjust Volumes",
-  "gui.voicechat.select_microphone.title": "Select Microphone",
-  "gui.voicechat.select_speaker.title": "Select Speaker",
-  "gui.voicechat.group.title": "Group Chat",
-  "gui.voicechat.join_create_group.title": "Join or Create Group",
+  "gui.voicechat.select_microphone.title": "Select a Microphone",
+  "gui.voicechat.select_speaker.title": "Select Speakers",
+  "gui.voicechat.group.title": "Group Management",
+  "gui.voicechat.join_create_group.title": "Join or Create a Group",
   "gui.voicechat.create_group.title": "Create a Group",
   "gui.voicechat.enter_password.title": "Enter Group Password",
   "message.voicechat.settings": "Settings",
   "message.voicechat.group": "Group",
-  "message.voicechat.voice_chat_volume": "Voice chat volume: %s",
-  "message.voicechat.microphone_amplification": "Microphone amplification: %s",
-  "message.voicechat.mic_test_unavailable": "Microphone testing unavailable",
-  "message.voicechat.mic_test.disabled": "Press to enable microphone testing",
-  "message.voicechat.mic_test.enabled": "Press to disable microphone testing",
+  "message.voicechat.voice_chat_volume": "Voice Chat Volume: %s",
+  "message.voicechat.microphone_amplification": "Microphone Amplification: %s",
+  "message.voicechat.mic_test_unavailable": "Microphone testing unavailable.",
+  "message.voicechat.mic_test.disabled": "Press to enable microphone testing.",
+  "message.voicechat.mic_test.enabled": "Press to disable microphone testing.",
```

Comment:
Ending sentences in tooltips with a period just makes sense, especially when they describe or explain something. Just like in normal written language, you end sentences with a period, so I don't see why it should be any different in Minecraft.

Mojang seems to have noticed this too, and ever since the introduction of the "fabulous" graphics option, they have consistently followed this convention when adding new descriptions. Now, even short and simple sentences like "Classic stereo sound." end with a period, which, in this particular case, I find a bit comical.

As for the chat messages, I decided not to add any periods there (and I didn't remove the ones you already added either) since Mojang seems to prefer leaving them out in this case. They might change their mind in the future, though.

Reply:
Thats fair enough